scripts/release.sh

```diff
@@ -53,6 +53,10 @@ script_check=1
 mainline=1
 channel=mainline
 
+# These values will be used for any PRs created.
+pr_review_assignee=${CODER_RELEASE_PR_REVIEW_ASSIGNEE:-@me}
+pr_review_reviewer=${CODER_RELEASE_PR_REVIEW_REVIEWER:-bpmct,stirby}
```

Comment:
nit: maybe add yourself as reviewer too in case of emergency

Reply:
I don’t want to involve myself too much in the process, since this is just docs changes I don’t think I’m needed. I will cooperate with @stirby during the next release(s) to ensure the process is smooth.